doc/manual.adoc

```diff
@@ -175,6 +175,7 @@ following variables.
 			 "." if undefined.
 |%(file)		|The currently selected file.
 |%(lineno)		|The currently selected line number. Defaults to 0.
+|%(lineno_old)		|The selected line number in the pre-image of a diff. Defaults to 0.
```

Comment:
Just wondering if we can't find a better name for lineno_old. Not that easy though...

Reply:
Maybe `lineno_original`, that's what Wikipedia's diff page uses.

When browsing a patch that has not yet been applied to the working tree, this line number can be used to go to the exact line under the cursor. Of course this can only work if the cursor is on a (future) deleted, or context line. If the cursor is in a sequence of added lines, this line number will be on the first of the sequence.

I briefly considered making it symmetrical by adding an alias, so we can have something like `lineno_before` and `lineno_after` bur that's probably, not worth it.

Reply:
So be it for lineno_old, it not worth the time, you're right.

Reply:
`doc/tigrc.5.adoc` should be impacted as well.

Reply:
"old" seems alright because `git-diff` uses this terminology - it has options like `--output-indicator-old`.

`pre-image` was bogus though. It implies a function / unique mapping, which doesn't make sense here. I think I got that from the message that rerere prints ("Recorded preimage ...").

I've updated the docs and commit message